### ISSUE TITLE
err:   D3D11CreateDevice: Failed to create a DXGI factory

### ISSUE THREAD

**Issue by Heus-Sueh** [doitsujin/dxvk#3188]
Please describe your issue as accurately as possible.

If you use Windows, please check the following page: https://github.com/doitsujin/dxvk/wiki/Windows

### Software information
Using Bottles to run DARK SOULS REMASTERED

### System information
- GPU: GTX 1060 6GB
- Driver: 525.60.11
- Wine version: 7.0
- DXVK version: v2.0-59-g196fefec

### Apitrace file(s)
- Put a link here

For instructions on how to use apitrace, see: https://github.com/doitsujin/dxvk/wiki/Using-Apitrace

### Log files
- d3d9.log: 
- d3d11.log:
[DarkSoulsRemastered_d3d11.log](https://github.com/doitsujin/dxvk/files/10420504/DarkSoulsRemastered_d3d11.log)
- dxgi.log:
[DarkSoulsRemastered_dxgi.log](https://github.com/doitsujin/dxvk/files/10420506/DarkSoulsRemastered_dxgi.log)

I believe that this error occurs due to the lack of some dependency but I don't know which dependency that would be, I already tried to reinstall the driver but the error persists, what are the possible solutions for this?

**Comment by mbriar** [comment]
If this is plain wine 7.0, then winevulkan is too old to run DXVK 2.0 and newer. Either update wine or use DXVK 1.10.3.

**Comment by Heus-Sueh** [comment]
> If this is plain wine 7.0, then winevulkan is too old to run DXVK 2.0 and newer (see https://github.com/doitsujin/dxvk/wiki/Driver-support). Either update wine or use DXVK 1.10.3.

2.0 and 1.10.3 doesn't work for me either

**Comment by mbriar** [comment]
Then there must be some other setup issue going on. That error message you get pretty much means that there is no vulkan driver available.

15 Jan 2023, 16:46 by ***@***.***:

**Comment by Blisto91** [comment]
Yes as mbriar said according to your dxgi log it seems there is a driver issue. Make sure your vulkan drivers are installed and working correctly

**Comment by Heus-Sueh** [comment]
what is the package names of the vulkan drivers in ubuntu?, i'm pretty sure i have them installed

**Comment by mbriar** [comment]
If you run this from flatpak somehow then I'd guess that the nvidia drivers from the flatpak runtime don't work with your host nvidia drivers or are broken for other reasons. And just to be clear DXVK 2.0 will not work with wine 7.0, so best try on DXVK 1.10.3.

15 Jan 2023, 17:02 by ***@***.***:

**Comment by Heus-Sueh** [comment]
this bug is giving me a headache, i can't find the solution

**Comment by Blisto91** [comment]
I totally missed you are using bottles. I guess that means flatpak, make sure that your system drivers and the flatpak drivers match each other. You try updating your flatpaks `flatpak update`

**Comment by Heus-Sueh** [comment]
> I totally missed you are using bottles. I guess that means flatpak, so make sure that your system drivers and the flatpak drivers match each other. You can try updating your flatpaks `flatpak update`

no update for flatpaks and nvidia drivers are working normally, 
![Screenshot from 2023-01-15 13-52-48](https://user-images.githubusercontent.com/98482469/212554962-fab8a7c1-ee3c-4f02-9d7d-fd654df63189.png)


but it seems that dxvk is not finding what it needs
![Screenshot from 2023-01-15 12-08-35](https://user-images.githubusercontent.com/98482469/212554952-52867529-8bfd-494c-8fa5-614090f7884b.png)

**Comment by Leopard1907** [comment]
That was not about flatpak package update but rather for runtime. So make sure there is no mismatch in runtime and host.

Also do you have Vulkan drivers installed?

`sudo apt install libvulkan1 libvulkan1:i386`

**Comment by Heus-Sueh** [comment]
> That was not about flatpak package update but rather for runtime. So make sure there is no mismatch in runtime and host.
> 
> Also do you have Vulkan drivers installed?
> 
> `sudo apt install libvulkan1 libvulkan1:i386`

shows that it is installed
![image](https://user-images.githubusercontent.com/98482469/212556446-6fc3b5f4-8d2d-4e97-b919-38c9d995e1ac.png)

**Comment by Heus-Sueh** [comment]
> That was not about flatpak package update but rather for runtime. So make sure there is no mismatch in runtime and host.

how do i see if there is runtime and host mismatch?

**Comment by Blisto91** [comment]
I don't use flatpak but i think you can probably find your flatpak installed version with `flatpak list`
